es-ds-components/components/es-collapse.vue

```diff
@@ -85,3 +85,12 @@ const onClick = ({ value }: { value: boolean }) => {
         <slot />
     </panel>
 </template>
+<style lang="scss" scoped>
+@use '@energysage/es-ds-styles/scss/variables' as variables;
+
+:deep(.es-collapse-toggler:focus-visible) {
+    border: 0;
```

Comment:
Overall looks good but I'd remove this `border: 0;` line, as it's causing the contents of the collapse to shift upon receiving focus - the outline should be enough to achieve the look you want.